hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1388,6 +1379,25 @@
     </description>
   </property>
 
+  <property>
+    <name>hdds.scm.safemode.pipeline.creation</name>
+    <value>true</value>
+    <tag>HDDS,SCM,OPERATION</tag>
+    <description>
+      Boolean value to enable background pipeline creation in SCM safe mode.
+    </description>
+  </property>
+
+  <property>
+    <name>hdds.scm.safemode.min.pipeline</name>
```

Comment:
Same as mentioned earlier, can we consolidate the criteria for pipeline to exit safemode?

Reply:
When a new cluster startup, it will exit safemode immediately currently.  Because there is pipeline to  wait for it's report from datanode.   So if we don't add a min pipeline threshold, we will exit safemode without any pipeline ready to use.  
The desired state is when safenode off, Ozone should be ready for read/write object. 
Thant's the reason this property is added.  HDDS_SCM_SAFEMODE_HEALTHY_PIPELINE_THRESHOLD_PCT_DEFAULT cannot fulfill this purpose, because the total pipeline number at the start of a new cluster is 0.